src/Pages/Auth/Login.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import { useFormik } from "formik";
+import { Link } from "react-router-dom";
+import * as Yup from "yup";
+import { TimelineLite, Power3 } from "gsap";
+import { makeAuthenticatedApiCall } from "../Api.js";
+
+const url = "https://atumaatu.herokuapp.com/v1/sessions/";
+
+const Login = (props) => {
+  const [loading, setLoading] = useState(false);
+  const [user, setUser] = useState(null);
+  const [authenticationError, setAuthenticationError] = useState(null);
+
+  if (user) {
+    props.history.push("/dashboard");
+    localStorage.setItem("user", JSON.stringify(user.token));
+  }
+  let tl = new TimelineLite();
+
+  useEffect(() => {
+    tl.from(".login", 3, { y: 40, ease: Power3.easeOut });
+  }, []);
+
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+      password: "",
+    },
+
+    validationSchema: Yup.object({
+      email: Yup.string()
+        .matches(
+          /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+          "Email must be valid"
+        )
+        .required("Required"),
+
+      password: Yup.string()
+        .min(6, "Password is too short")
+        .required("Required"),
+    }),
+
+    onSubmit: (values) => {
+      setLoading(true);
+      const responses = makeAuthenticatedApiCall("post", url, {
```

Comment:
So this part will now be

```suggestion
      const responses = makeAuthenticatedApiCall("post", path, {
```